java/libphonenumber/src/com/google/i18n/phonenumbers/CountryCodeToRegionCodeMap.java

```diff
@@ -30,7 +30,7 @@
   // country/region represented by that country code. In the case of multiple
   // countries sharing a calling code, such as the NANPA countries, the one
   // indicated with "isMainCountryForCode" in the metadata should be first.
-  static Map<Integer, List<String>> getCountryCodeToRegionCodeMap() {
+  public static Map<Integer, List<String>> getCountryCodeToRegionCodeMap() {
```

Comment:
Can we have a public getter instead? Maybe we will be breaking some clients that creates duplicate packages / using reflections to get this. Please let me know if I am wrong/missing something. Just checking as it is not part of lib objectives to provide this mapping.

Reply:
I'm not sure I understand what you mean. This **is** already a public getter :)

Reply:
Sorry... thought we are changing scope of member...